src/sub-form-accessor.ts

```diff
@@ -25,6 +25,10 @@ export class SubFormAccessor<
     // no op
   }
 
+  async validate(): Promise<boolean> {
+    return this.errorValue === undefined;
```

Comment:
This doesn't look right: a sub form is also invalid if any of its fields or other sub-accessors is invalid. Why did you need to add this?

Reply:
I added this to determine the validity of the subForm. Without it, the validate function form state.ts would not kick this off.

You do have a point that the subform is also invalid if some of its sub-accessors are. I will ad them to this function